tests/Feature/BookTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Tests\Feature;
+
+// use Illuminate\Foundation\Testing\RefreshDatabase;
+use App\Models\Book;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\TestCase;
+use Illuminate\Testing\Fluent\AssertableJson;
+
+class BookTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    /**
+     * A basic test example.
+     */
+    public function test_getAllBooks_success(): void
+    {
+        Book::factory()->count(10)->create();
+
+        $response = $this->get('/api/books');
+
+        $response->assertStatus(200)
+            ->assertJson(function (AssertableJson $json) {
+                $json->hasAll(['data', 'message'])
+                    ->has('data', 10, function (AssertableJson $json) {
+                        $json->hasAll([
+                            'id',
+                            'title',
+                            'author',
+                            'image',
+                            'genre'
```

Comment:
This test is pretty good, only problem is that it doesn't assert anything about what's in genre. You should really assert that genre contains the right info as well